packages/sku/telemetry/index.js

```diff
@@ -1,24 +1,32 @@
 const os = require('node:os');
 
-const { getPathFromCwd } = require('../lib/cwd');
+const { getPathFromCwd, cwd } = require('../lib/cwd');
 const isCI = require('../lib/isCI');
 const provider = require('./provider');
 const skuVersion = require('../package.json').version;
 
 const { languages } = require('../context');
 
 let projectName = 'unknown';
+let braidVersion = 'unknown';
 try {
   const packageJson = require(getPathFromCwd('package.json'));
 
   if (packageJson.name) {
     projectName = packageJson.name;
   }
+
+  const braidPackageJson = require(require.resolve(
+    'braid-design-system/package.json',
+  ), [cwd()]);
+
+  braidVersion = braidPackageJson.version;
 } catch (e) {}
```

Comment:
Can we add a `require('debug')('sku:telemetry')` message here?

Reply:
For the `catch`? Sure. The telemetry package already spits out debug logs for the stuff it handles. https://github.com/SEEK-Jobs/sku-telemetry/pull/17

Reply:
I meant on the sku side, so we see the error if any of the `package.json`s can't be loaded.